.evergreen/serverless/create-instance.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
My understanding is that this is more portable. It's consistent with what @eramongodb did in other recent PRs.

Reply:
That is the intention, yes. See https://unix.stackexchange.com/q/29608 for a deeper discussion on its merits and drawbacks.